core/trino-main/src/main/java/io/trino/sql/planner/ExpressionInterpreter.java

```diff
@@ -757,18 +757,23 @@ protected Object visitArithmeticUnary(ArithmeticUnaryExpression node, Object con
                     if (handle.type().parameterCount() > 0 && handle.type().parameterType(0) == ConnectorSession.class) {
```

Comment:
Please ignore this commit, I'll remove it later.
I'm using eclipse and ecj doesn't like yield within try for some reason. M2e also errors out on missing property.